platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/components/EventIntake.java

```diff
@@ -64,30 +70,52 @@ public class EventIntake {
     /** Stores events, expires them, provides event lookup methods */
     private final ShadowGraph shadowGraph;
 
+    private final ExecutorService prehandlePool;
+    private final Consumer<EventImpl> prehandleEvent;
+
     /**
      * Constructor
      *
-     * @param selfId                        the ID of this node
-     * @param consensusSupplier             a functor which provides access to the {@code Consensus} interface
-     * @param addressBook                   the current address book
-     * @param dispatcher                    an event observer dispatcher
+     * @param platformContext   the platform context
```

Comment:
Prefacing all my style nits now with the caveat "ignore if you wanna":

- javadocs here are out of order
- no javadoc was added for `prehandleEvent`

Reply:
fixed